_rules/additional-content-triggered-on-hover-is-hoverable-ep1s13.md

```diff
@@ -21,22 +21,17 @@ acknowledgments:
 
 ## Applicability
 
-The rule applies to any element that when [hovered][] causes [visible changes in content][].
+The rule applies to any element that when [hovered][] causes [visible changes in content][] and the [area affected by the changes][area affected by visible changes] is adjacent or overlaps the target content.
```

Comment:
Sorry, but what does "adjacent or overlaps" mean? I think we need to define that in terms of bounding box positions.

Reply:
When writing it, I was wondering if I should do that. Guess that I should have. Will do it now.

Reply:
I'll take the opportunity to add another inapplicable example illustrating this new condition.